src/nn.h

```diff
@@ -32,6 +32,17 @@ extern "C" {
 #include <errno.h>
 #include <stddef.h>
 
+/*  64-bit integer type for statistics.  */
+#if defined(_WIN32)
+typedef	 __uint64 NN_UINT64
```

Comment:
In modern MSVC (2010 edition or later) this is true.  Older versions don't have it.  I fixed it for now, but I hate this.  If we can just state that MSVC 2010 or newer is required, that would be simpler.  (See also the __VMS hackery for OpenVMS.  Puke.)  What I don't know is whether requiring MSVC 2010 or newer will cut off an additional round of platforms (Vista or Windows 7) that would otherwise be supported.  I have tried (mostly successfully) remain blissfully unaware of Microsoft platforms up until now...